athena-jdbc/src/main/java/com/amazonaws/connectors/athena/jdbc/saphana/SAPHANARecordHandler.java

```diff
@@ -0,0 +1,94 @@
+/*-
+ * #%L
+ * athena-jdbc
+ * %%
+ * Copyright (C) 2019 Amazon Web Services
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+package com.amazonaws.connectors.athena.jdbc.saphana;
+
+import com.amazonaws.athena.connector.lambda.domain.Split;
+import com.amazonaws.athena.connector.lambda.domain.TableName;
+import com.amazonaws.athena.connector.lambda.domain.predicate.Constraints;
+import com.amazonaws.connectors.athena.jdbc.connection.DatabaseConnectionConfig;
+import com.amazonaws.connectors.athena.jdbc.connection.GenericJdbcConnectionFactory;
+import com.amazonaws.connectors.athena.jdbc.connection.JdbcConnectionFactory;
+import com.amazonaws.connectors.athena.jdbc.manager.JDBCUtil;
+import com.amazonaws.connectors.athena.jdbc.manager.JdbcRecordHandler;
+import com.amazonaws.connectors.athena.jdbc.manager.JdbcSplitQueryBuilder;
+import com.amazonaws.services.athena.AmazonAthena;
+import com.amazonaws.services.athena.AmazonAthenaClientBuilder;
+import com.amazonaws.services.s3.AmazonS3;
+import com.amazonaws.services.s3.AmazonS3ClientBuilder;
+import com.amazonaws.services.secretsmanager.AWSSecretsManager;
+import com.amazonaws.services.secretsmanager.AWSSecretsManagerClientBuilder;
+import com.google.common.annotations.VisibleForTesting;
+import org.apache.arrow.vector.types.pojo.Schema;
+import org.apache.commons.lang3.Validate;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+
+/**
+ * Data handler, user must have necessary permissions to read from necessary tables.
+ */
+public class SAPHANARecordHandler
```

Comment:
naming convention

Reply:
same as above

Reply:
Thats fine, choose a name that uses CammelCase

Reply:
Fixed, please check